src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Promotion.xml

```diff
@@ -36,23 +37,35 @@
 
         <attribute name="description">
             <group>admin:promotion:read</group>
+            <group>admin:promotion:create</group>
+            <group>admin:promotion:update</group>
         </attribute>
 
         <attribute name="priority">
             <group>admin:promotion:read</group>
+            <group>admin:promotion:create</group>
+            <group>admin:promotion:update</group>
+            <denormalization_context>
+                <entry name="disable_type_enforcement">true</entry>
```

Comment:
Why are you disabling it on the priority field?

Reply:
In the database, the priority column is not nullable, so serialization throws an error on null values. However, we want to allow sending null, because the `setPriority` method accepts null.

Reply:
IMO, if the database doesn't accept the null value, we shouldn't accept it during serialization

Reply:
![image](https://github.com/Sylius/Sylius/assets/33687392/9d3a96eb-552c-4927-8f17-623ca5aa4469)
Setter allows setting a null value. If I set the value to null, it receives the last priority. If we remove the ability to set null, we would know what the last priority is.

Reply:
#15551 I have removed possibility to set null value. We can achieve the same goal to pass a value of minus one.